src/routing/core.rs

```diff
@@ -1108,7 +1108,7 @@ impl Core {
             };
             if recipients.is_empty() {
                 trace!("Cannot route user message, recipient list empty: {:?}", msg);
-                return Err(Error::CannotRoute);
+                return Err(Error::EmptyRecipientList);
```

Comment:
I do think the previous `CannotRoute` error back is more accurate?

Reply:
I felt `EmpyRecipientList` gives a more clearer image of the error compared to `CannotRoute`